src/components/MainView.js

```diff
@@ -38,7 +51,11 @@ function MainView({ children }) {
           flex-shrink: 0;
         `}
       >
-        <Header compactMode={compactMode} toggleMenuPanel={toggleMenuPanel} />
+        <Header
```

Comment:
Ahh, something I also just noticed, we should remove the bit of padding or margin separating the activity panel with the preferences panel surface:

<img width="116" alt="Screen Shot 2020-04-08 at 3 25 39 PM" src="https://user-images.githubusercontent.com/4166642/78789453-722bd600-79ad-11ea-9777-cb9a8f9e5f19.png">

(The orange higlight)